plugins/backend-serverless-app/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import CircleCiConfigHook, {
+  CircleCIStatePartial,
+  generateConfigWithJob
+} from '@dotcom-tool-kit/circleci/lib/circleci-config'
+import { ToolKitError } from '@dotcom-tool-kit/error'
+import { getOptions } from '@dotcom-tool-kit/options'
+
+export class ServerlessDeployReview extends CircleCiConfigHook {
```

Comment:
**question:** Why is this hook defined in here rather than the `@dotcom-tool-kit/serverless` plugin?

Reply:
I'm also interested in the answer to this. I was going to reply to Joel that I thought it was because we needed to import the Circle plugin in order to override the hook and that maybe made more sense at the app level, but actually it could happen in the Serverless plugin too

Reply:
No that's right, Jen! The Serverless plugin is generic and shouldn't be aware of CircleCI so that we could support other CI systems in the future. The `backend-serverless-app` is where the serverless and circleci-deploy plugins get pulled in together so it makes sense to put the CircleCI hook for Serverless code there.

Reply:
We could handle it so that we check for the `@dotcom-tool-kit/serverless` options in the `circleci-deploy` plugin but this feels a little weird to me. Though maybe it's okay? I'll try it out

Reply:
👍 I'm trying to think about it in terms of a new developer coming to Tool Kit in a year's time and it being as obvious and predictable as possible why things happen where they do. I don't know that my suggestion is the better solution but I appreciate you considering it and having a go!